### DISCUSSION TITLE
Is there a way to publish conductor metrics to AWS Cloudwatch using spectator?

### DISCUSSION THREAD

**Discussion by vamsibandarupalli** [Netflix/conductor#2724] [Q&A]
Hi, I can see conductor-contribs has default logging metrics, but is there a way that we can publish these metrics to AWS cloudwatch?

**Comment by apanicker-nflx** [comment]
I think this question would be better answered by [Netflix/spectator](https://github.com/Netflix/spectator) and Conductor provides integration with Spectator for metric collection. [Other metric collection libraries](https://github.com/Netflix/conductor/tree/main/contribs/src/main/java/com/netflix/conductor/contribs/metrics) that are also integrated are Prometheus and Datadog.
